### DISCUSSION TITLE
Code in Documents doesn't work

### DISCUSSION THREAD

**Discussion by TQuy** [twbs/bootstrap#32581] [Q&A] [answered]
I try to recreate things from https://getbootstrap.com/docs/5.0/layout/columns/ by adding the code into "Stater template" and other templates found in  https://getbootstrap.com/docs/5.0/examples/  but it doesn't work. It makes me frustrated because this is not the first time I've met this problem and the examples are supposed to be simple

**Comment by mdo** [comment, marked as answer]
The red boxes and styling won't come with it because that's from our docs CSS. The rest should work just fine though. We can't help much beyond that if you're not showing us a live example of the problem though.

**Reply by TQuy** [reply]
This is the code 
![image](https://user-images.githubusercontent.com/45731304/102852211-4d4e9480-4450-11eb-8781-23f7345f2391.png)

(I cannot paste the HTML code)

![image](https://user-images.githubusercontent.com/45731304/102852079-006abe00-4450-11eb-88ee-01120870a0e1.png)

**Reply by TQuy** [reply]
Okay I get the problem, It is just because I don't have the css stylesheet, thanks
